graphics/libjpeg-turbo/Portfile

```diff
@@ -46,8 +47,11 @@ if {${universal_possible} && [variant_isset universal]} {
 } elseif {${configure.build_arch} in {i386 x86_64}} {
     depends_build-append port:nasm
     configure.env       ASM_NASM=${prefix}/bin/nasm
+} elseif {${configure.build_arch} eq "ppc"} {
+    if {[catch {sysctl hw.vectorunit} result] || $result == 0} {
+        configure.args-append   -DWITH_SIMD=OFF
```

Comment:
Also needs to go in `merger_configure_args(ppc)` in the universal case.

Reply:
Can you clarify this comment or point me to an example?